README.md

```diff
@@ -49,7 +49,25 @@ sed -i "s/<REPLACE ME>/GITHUB_NICK/g" boxes.d/99-local.yaml
 vagrant up centos7-devel
 ```
 
-You can find more thorough guides in the [docs folder](https://github.com/theforeman/forklift/tree/master/docs).
+In case using vagrant is not desired, ansible playbooks and roles from this repo can be used separately. This is useful if an existing host should be used for the installation, e.g. a beaker machine. In order to deploy the devel environment on host test.example.com, the following needs to be done:
+
+on test.example.com machine, where the dev env should be deployed
+```sh
+useradd vagrant
+echo "vagrant	ALL=(ALL)	NOPASSWD: ALL" >> /etc/sudoers`
+```
+
+in forklift checkout
+```sh
+echo "[devel]\ntest.example.com" > inventories/local_inventory
+ansible-playbook --private-key=~/.ssh/id_rsa --user root --inventory inventories/local_inventory --extra-vars katello_devel_github_username=katello playbooks/devel.yml
```

Comment:
* https://github.com/theforeman/forklift/commit/74cfc77aae71cdb400185698f9ed557cc847e5f7 may make ```--private-key``` redundant.
* Since you created the vagrant user with sudo, can you use ```--become``` instead of ```--user root```? Can you leave it out altogether since ```playbooks/devel.yml``` has ```become: true```?
* Does 1051966ca9d2ffc06640ad69f0b0e472b87f8139 make ```--inventory``` redundant or is good to leave it in because it might deploy the devel playbook to your vagrant hosts?

Reply:
the first 2 bullets:
I can add additional example of how to specify different private ssh key, but this example is for how to use it against any host. Therefore I find example of how to specify custom credentials useful, in this case default ssh key and default root but still.

last:
I don't understand how that would work if I want to specify arbitrary host name, that's the point of line 62. Or is `inventories/local_inventory` now default inventory? It was not clear from the linked commit to me.